composer.json

```diff
@@ -36,7 +36,8 @@
         "symfony/phpunit-bridge": "^5.0",
         "symfony/stopwatch": "5.0.*",
         "symfony/var-dumper": "5.0.*",
-        "symfony/web-profiler-bundle": "5.0.*"
+        "symfony/web-profiler-bundle": "5.0.*",
+        "symfony/web-server-bundle": "4.4.5"
```

Comment:
The WebServerBundle is not recommended anymore: use the Symfony Local Server instead :) (https://symfony.com/doc/current/setup/symfony_server.html). It's even integrated with Docker, helping you setup even more efficiently the project.

Reply:
Here is what I get when I try the `symfony` CLI:
```
$ symfony server:start
[InvalidArgumentException]
There are no commands defined in the "server" namespace.
```

Do you use Docker yourself ? Or the symfony CLI ?

Reply:
Ah, it's a mistake in the doc! It should be `symfony serve -d` :) . I use both Docker (for web services) and Symfony CLI (for PHP): much faster on non-linux platforms and allows me to easily have SSL in local :) .